setup.cfg

```diff
@@ -26,4 +28,7 @@ ignore =
 # E251: Unexpected spaces around keyword / parameter equals (types in function definitions)
 # E402: module level import not at top of file
 # E999: Internal AST compilation error (flake8 specific)
-per-file-ignores = *.pyx: E211,E225,E226,E227,E251,E402,E999
+per-file-ignores = 
+    *.pyx: E211,E225,E226,E227,E251,E402,E999
+    apache_json.py: E226,E501
```

Comment:
It's better to add the description for the newly added error names above, around line 30, while respecting the order.